server/controllers/stock/core.js

```diff
@@ -731,6 +731,12 @@ function processMultipleLots(inventories) {
         lot.S_RISK_QUANTITY = Math.round(lot.S_RISK * lot.avg_consumption);
         lotLifetime += lot.S_LOT_LIFETIME;
 
+        const numMonthsOfStockLeft = (lot.quantity / lot.CMM); // how many months of stock left
+        const today = new Date();
+        // if we have more months of stock than the expiration date,
+        // then we'll need to label these are in risk of expiration
+        const isInRiskOfExpiration = lot.expiration_date < (today.setMonth(today.getMonth() + numMonthsOfStockLeft));
```

Comment:
I know the issue had this pseudo-code in it, but now that I look at it, it might make more sense to make this calculate use days of stock, rather than months.  Stock expired by 2 days isn't able to be sold.

So, let's do something like:
```js
const numMonthsOfStockLeft = (lot.quantity / lot.CMM);
const numDaysOfStockLeft =  numMonthsOfStockLeft * 30.5;
const  isInRiskOfExpiration = lot.expiration_date <  moment(today).add(numDaysOfStockLeft, 'days').toDate();
```